tests/unit/test_modulegraph/test_modulegraph.py

```diff
@@ -467,6 +467,8 @@ def assertNoMethods(self, klass):
         if '__dict__' in d:
             # New in Python 3.4
             del d['__dict__']
+        if '__weakref__' in d:
+            del d['__weakref__']
```

Comment:
Please add a more more information to the commit message, e.g. "remove weakref, which should not be considered when comparing the attributes".